packages/localize-tools/src/tests/e2e/extract-xliff-fresh-ph.test.ts

```diff
@@ -0,0 +1,12 @@
+/**
+ * @license
+ * Copyright 2020 Google LLC
```

Comment:
Here and elsewhere: 2021

Reply:
This is just a rename, so it's copyright date shouldn't change (right?). And the other new files are also clones.